### PR TITLE
Added autocomplete for trigger related features

### PR DESCRIPTION
**Pull request by sluongng**
Here I accomplished 2 things:

- Added autocomplete related to `fcli trigger` command based on information in `fcli trigger help` and `fcli trigger <command> --help`

- Added `nodejs8` runtime type into autocompletion

Please help review and merge

### CODE REVIEW

**Comment by sluongng** [comment]
@muxiangqiu fixed

File: misc/completion/_fcli [hunk 1]
```diff
@@ -33,6 +42,7 @@ _1st_arguments=(
 	'help:Help about any command'
 	'service:service related operation'
 	'shell:interactive shell'
+	'trigger: trigger related operation'
```
Comment (muxiangqiu) [inline misc/completion/_fcli]: _fcli_help_args need to add "trigger: trigger related operation"

File: misc/completion/fcli-completion.bash [hunk 2]
```diff
@@ -311,6 +338,119 @@ function _fcli() {
...
+					get)
+						local opts
+						case $prev in 
+							--service-name|-s)
+								opts="$(__fcli_get_all_service_name)"
+								;;
+							--function-name|-f)
+								opts="$(__fcli_get_all_function_name)"
+								;;
+							--trigger-name|-f)
```
Comment (muxiangqiu) [inline misc/completion/fcli-completion.bash]: -f is changed to -t

File: misc/completion/fcli-completion.bash [hunk 3]
```diff
@@ -311,6 +338,119 @@ function _fcli() {
...
+					update)
+						local opts
+						case $prev in 
+							--service-name|-s)
+								opts="$(__fcli_get_all_service_name)"
+								;;
+							--function-name|-f)
+								opts="$(__fcli_get_all_function_name)"
+								;;
+							--trigger-name|-f)
```
Comment (muxiangqiu) [inline misc/completion/fcli-completion.bash]: -f is changed to -t

File: misc/completion/fcli-completion.bash [hunk 4]
```diff
@@ -311,6 +338,119 @@ function _fcli() {
...
+					delete)
+						local opts
+						case $prev in 
+							--service-name|-s)
+								opts="$(__fcli_get_all_service_name)"
+								;;
+							--function-name|-f)
+								opts="$(__fcli_get_all_function_name)"
+								;;
+							--trigger-name|-f)
```
Comment (muxiangqiu) [inline misc/completion/fcli-completion.bash]: -f is changed to -t

**Comment by sluongng** [comment]
@muxiangqiu please help review again 🙇

**Pull request merged**